Pod/Classes/VOKManagedObjectMapper.m

```diff
@@ -190,6 +190,11 @@ - (Class)expectedClassForObject:(NSManagedObject *)object andKey:(id)key
 
         Class managedObjectClass = objc_getClass(className);
         objc_property_t prop = class_getProperty(managedObjectClass, propertyName);
+        if ([self isKindOfClass:[VOKManagedObjectDefaultMapper class]]) {
```

Comment:
If this is run in a release environment with assertions turned off, will this empty if statement be optimized out by the compiler?  If not, does it make sense to refactor this to make sure it doesn't do this check in release builds?